### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by deetz99**
update all dependencies in main app, updated everything to latest except for eslint, breaks with plugins

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-264-9wocrmhr.web.app

File: web/site/nuxt.config.ts [hunk 1]
```diff
@@ -2,18 +2,36 @@
 export default defineNuxtConfig({
   devtools: { enabled: false },
   ssr: true,
+
```
Comment (deetz99) [inline web/site/nuxt.config.ts:5]: The new nuxt version generates as a SPA, so now we must explicitly set what routes to prerender for nuxt content to generate correctly

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-264-9wocrmhr.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**